src/site/content/en/blog/when-to-use-local-https/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: When to use HTTPS for local development
+subhead:
+authors:
+  - maudn
+date: 2020-12-07
+updated: 2020-12-07
+hero: hero.jpg
+tags:
+  - blog
+  - security
+---
+
+_In this article, statements about `localhost` are valid for `127.0.0.1` and `[::1]` as well, since they both describe the local computer address, also called "loopback address". Also, to keep things simple, the port number isn't specified._
+_So when you see `http://localhost`, read it as `http://localhost:{port}` or `http://127.0.0.1:{port}`._
+
+## Summary
+
+When developing locally, use `http://localhost` by default. Service Workers, web authentication APIs, and more, will work.
+But in some cases, you'll need HTTPS:
+
+- Secure cookies need HTTPS.
+- Mixed-content debugging needs HTTPS.
+- HTTP/2 and later needs HTTPS.
+- Custom hostnames need HTTPS.
+
+  <figure class="w-figure">
+    <img src="./when-to-use-local-https.jpg" alt="List: when to use HTTPS for local development">
+    <figcaption class="w-figcaption">When to use HTTPS for local development</figcaption>
+  </figure>
+
+See [How to use HTTPS for local development](/how-to-use-local-https).
+
+✨ This is all you need to know. Keep reading for more details!
+
+## Why your development site should behave securely
+
+You want your local development site to behave as much as possible like your production website, to avoid running intro unexpected issues.
```

Comment:
```suggestion
To avoid running intro unexpected issues, you want your local development site to behave as much as possible like your production website.
```